code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -255,10 +255,10 @@
 	lefthand_file = 'icons/mob/inhands/equipment/custodial_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/custodial_righthand.dmi'
 	custom_materials = list(/datum/material/iron=200)
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
 	amount_per_transfer_from_this = 20
-	possible_transfer_amounts = list(5,10,15,20,25,30,50,70)
-	volume = 70
+	possible_transfer_amounts = list(5,10,25,50,100,200)
+	volume = 200
```

Comment:
![image](https://user-images.githubusercontent.com/34697715/169640333-94b8d5db-0d6b-4516-8b4b-1ed7597b71c8.png)

Reply:
excuse me wut

Reply:
I think it is a fair value for having to carry them by hand when you are moving from one department to another.
Water wasn't an issue if people did choose to carry extra water in a beaker or two to refill the bucket on the go.

Now if there is something I'm missing with the value please tell me.